courses/machine_learning/images/mnist_linear.ipynb

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
<p>I'd add --port 8080 to make clear on which port to connect. The first time I ran this cell I got confused since it blocks the notebook execution. This is fine though because you can run the cell, look at tensorboard on localhost:8080, then interupt the cell to continue for the next cells. </p><p>Maybe add some explanation around that workflow.</p><p> </p><p>Also: I am testing the notebook locally. Does it work on dlvm? (localhost should be changed to something else up there, the port should be open also to receive connections and not blocked by firewall rules, right?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Ah, good point. I updated the instructions. Are they clearer?
Yes, I'm building and testing this code on a dlvm, which expects to use port 6006. Does that work for you locally?